contracts/src/CrossChainApplications/NativeTokenBridge/INativeTokenDestination.sol

```diff
@@ -0,0 +1,45 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: Ecosystem
+
+pragma solidity 0.8.18;
+
+/**
+ * @dev Interface that describes functionalities for a cross-chain native token bridge.
```

Comment:
nit: think we might want to avoid `bridge` for confusion, and could specify the description for destination, so around minting and burning native assets.

Reply:
same for other comments about bridging

Reply:
Updated these comments. Let me know if they make more sense.